index.html

```diff
@@ -3,9 +3,41 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Rylie Butner | Portfolio</title>
 </head>
 <body>
-    <iframe width="560" height="315" src="https://www.youtube.com/embed/gdsDkEb1k94" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+    <header>
+        <nav>
+            <ul>
+              <li><a href="index.html"><h2>Rylie Butner</h2></a></li>
+              <li><a href="work.html">Work</a></li>
+              <li><a href="misc.html">Misc.</a></li>
+              <li><a href="contact.html">Contact</a></li>
+            </ul>  
+        </nav>
+    </header>
+    <main>
+        <div><img src="images/clip-art.jpg" alt="Me as clip art"></div>
+        <div>
+            <h4>Design | Librarianship</h4>
```

Comment:
I recommend using a span. We want to use headings and their appropriate levels to communicate the hierarchy of the content on the page. It is also an accessibility issue to skip heading levels. See: https://www.a11yproject.com/posts/how-to-accessible-heading-structure/

Reply:
Thank you, this makes a lot of sense! I was making the mistake on choosing to use headings based on the size of text in my design.